sample/samples.h

```diff
@@ -28,22 +30,23 @@
 /*
 * Prints the contents of an error to stdout
 */
-inline void print_error(const ds3_error* error) {
+void print_error(const ds3_error* error) {
```

Comment:
I had to remove the inline qualifier to get it to build for me - these weren't being seen by the other files.  I can put it back since its just an example and the other files are building correctly now.